lib/sip2/messages/patron_information.rb

```diff
@@ -23,10 +24,8 @@ def build_patron_information_message(uid:, password:, terminal_password: nil)
         ].join
       end
 
-      def handle_patron_information_response(response)
-        return unless sequence_and_checksum_valid?(response)
-
-        Sip2::PatronInformation.new response
+      def handle_response(response)
+        Sip2::PatronInformation.new response if response =~ /\A64/
```

Comment:
How about assigning the response codes to class variables so it's easier to understand at first glance instead of having a random number here.

Reply:
I have another branch in behind this one that does exactly that